content/workshops/pki_workshop/unit_lessons/unit_1.md

```diff
@@ -0,0 +1,61 @@
+# Unit 1: Installing Directory Server
+
+In this unit you will install [Red Hat Directory Server](https://www.redhat.com/en/technologies/cloud-computing/directory-server "Red Hat Directory Server"). Red Hat Directory Server provides a centralized directory service for an intranet, network, and extranet information. Directory Server integrates with existing systems and acts as a centralized repository for the consolidation of employee, customer, supplier, and partner information. Directory Server can even be extended to manage user profiles, preferences, and authentication. 
+
+Red Hat Directory Server is used as the backend for Certificate System. The setup script can be used interactively but we are going to use an automated install with pre-configured settings.
```

Comment:
Red Hat Directory Server is used as the backend for [the] Certificate System.

Reply:
Fixed